#! /bin/sh /usr/share/dpatch/dpatch-run
## 024_use_dev_urandom.dpatch
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: use /dev/urandom instead of /dev/random, backport from 1.3.0

@DPATCH@
--- 1.3.x/configure.in	2008/05/02 18:46:13	652871
+++ 1.3.x/configure.in	2008/05/02 18:55:47	652872
@@ -1959,7 +1959,7 @@
   if test "$apr_devrandom" = "yes"; then
     # /dev/random on OpenBSD doesn't provide random data, so
     # prefer /dev/arandom, which does; see random(4).
-    for f in /dev/arandom /dev/random /dev/urandom; do
+    for f in /dev/arandom /dev/urandom /dev/random; do
       if test -r $f; then
         apr_devrandom=$f
         rand=1
